Sming/SmingCore/Network/TcpConnection.cpp

```diff
@@ -347,7 +347,7 @@ void TcpConnection::closeTcpConnection(tcp_pcb *tpcb)
 
 void TcpConnection::flush()
 {
-	if (tcp->state == ESTABLISHED)
+	if (tcp && tcp->state == ESTABLISHED)
```

Comment:
I have rebased from l;atest dev and somehow it still sees this as modified although the modified version is already in develop